### PR TITLE
CON-4011 Managed identity SQL account

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public static class ServiceCollectionExtensions
     {
         public static IServiceCollection AddDasDistributedMemoryCache(this IServiceCollection services, IConfiguration configuration, bool isDevelopment)
         {
-            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.ProviderRegistrations).Get<ProviderRegistrationsSettings>().RedisConnectionString;
+            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.RedisConnectionSettings).Get<ProviderRegistrationsSettings>().RedisConnectionString;
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs:11]:
```suggestion
            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.RedisConnectionSettings).Get<RedisConnectionSettings>().RedisConnectionString;

```

Comment (cofaulco) [reply src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs:11]: also the RedisConnectionString property should be removed from `ProviderRegisteationSettings.cs`

**Review by cofaulco — CHANGES_REQUESTED** [review]
1 change

File: src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs]: done

**Review by cofaulco — APPROVED** [review]

**Pull request merged**